sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/image_limit_settings.py

```diff
@@ -150,7 +149,7 @@ def __eq__(self, other: object) -> bool:
     def __ne__(self, other: object) -> bool:
         """Check inequality between two ImageLimitSettings objects.
 
-        :param other: ImageLimitSettings object
```

Comment:
why not ImageLimitSettings?

Reply:
can you please provide context? I am not sure what API review comments were?

Reply:
ne may be used to compare with any object, not just image limit settings object.